public/Buttons/PranavKale03/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Button</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <a href="#">
+          <span>Button</span>
+        </a>
+        <a href="#">
```

Comment:
Now I can see the buttons.
Animation is the same for both buttons.
Remove the red colored one and the respective styles.

Reply:
is it okay now?